### ISSUE TITLE
Add an 'inspect' method for opcode.js

### ISSUE THREAD

**Issue by thofmann** [bitpay/bitcore#1054]
Would this make sense?

```javascript
/**
 * Will return a string formatted for the console
 *
 * @returns {String} Script opcode
 */
Opcode.prototype.inspect = function() {
  return '<Opcode: ' + this.toString() + ', hex: '+this.toHex()+', decimal: '+this.num+'>';
};
```

**Comment by maraoz** [comment]
+1!

**Comment by thofmann** [comment]
Closed by #1071
